R/write.reportProject.R

```diff
@@ -109,6 +109,13 @@ write.reportProject <- function(mif, mapping,
   map<-map2
   remove(map2)
 
+  #check for duplicate rows
+  if (length(rownames(map[duplicated(map),])) > 0) {
```

Comment:
Why is it not just `if(anyDuplicated(rownames(map))`? Is rownames not sufficient here to check? (duplicated and anyDuplicated are quite ressource intensive making it important to keep this test as simple as possible)

Reply:
This does not work. `anyDuplicated(rownames(map))` will always return 0 because the rownames are generated when reading in the mapping, i.e. the rownames are not part of the mapping.

Reply:
I see. Thanks for clarification